source/extensions/tracers/xray/daemon.proto

```diff
@@ -17,6 +17,9 @@ message Segment {
   string origin = 9;
   string parent_id = 6;
   google.protobuf.Struct aws = 10;
+  bool error = 11;
```

Comment:
Can you add some 1-liner comments explaining what each message is. I know there's a link to the documentation there, but it's still seems appropriate to have some information here.

Reply:
Done.